addons/udes_stock/models/stock_picking.py

```diff
@@ -122,8 +122,9 @@ def _compute_picking_quantities(self):
                 if not has_discrepancies and qty_done != qty_todo:
                     has_discrepancies = True
 
-                total_qty_done += qty_done
-                total_qty_todo += qty_todo
+                factor = move.product_uom.factor
```

Comment:
Can you add a comment here to give an example of the relation between `factor` units of measure?

Reply:
I had a call w/ @peterpratt13 about this, because I wasn't sure I was understanding what was happening here.

So odoo calculates on the move under `product_qty` the quantity _in the uom of the product_
e.g
```
Product: cherry, UoM: Dozen (12)
Move: for 5x cherry, using UoM (on the move): Half a dozen (6)
product_qty will be 2.5 (computed to reflect the quantity in the uom set on the product)
```
this goes in-line with what @ws-mps was saying about the quants being stored in the uom of the product, so odoo flattens it to something common (products uom) and then it can push that to the quant.

So i think this factor calculation can be removed and instead we can just use `move.product_qty`, and `move.quantity_done` without converting things again

Reply:
yeah - so basically , need to change a bit because everything is Uom, missed the Real Quantity field as it is the Real Quantity in that UoM, different to the product uom. But everything is UoM not some

Reply:
So this (u_total_quanitiy_done in a picking might read 3, 1 crane, 2 boxxes of 100000000 screws.